radicle-cli/src/commands/inspect.rs

```diff
@@ -0,0 +1,222 @@
+#![allow(clippy::or_fun_call)]
+use std::ffi::OsString;
+use std::path::{Path, PathBuf};
+use std::process::{Command, Stdio};
+use std::str::FromStr;
+
+use anyhow::{anyhow, Context as _};
+use chrono::prelude::*;
+use json_color::{Color, Colorizer};
+
+use radicle::git;
+use radicle::identity::project::{Doc, IdentityError, Untrusted};
+use radicle::identity::Id;
+use radicle::storage::{ReadRepository, ReadStorage, WriteStorage};
+
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
+pub const HELP: Help = Help {
+    name: "inspect",
+    description: "Inspect a radicle identity or project directory",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad inspect <path> [<option>...]
+    rad inspect <id>   [<option>...]
+    rad inspect
+
+    Inspects the given path or ID. If neither is specified,
+    the current project is inspected.
+
+Options
+
+    --id-only   Return the ID in simplified form
+    --payload   Inspect the object's payload
+    --refs      Inspect the object's refs on the local device (requires `tree`)
+    --history   Show object's history
+    --help      Print help
+"#,
+};
+
+#[derive(Default, Debug, Eq, PartialEq)]
+pub struct Options {
+    pub path: Option<PathBuf>,
+    pub id: Option<Id>,
+    pub refs: bool,
+    pub payload: bool,
+    pub history: bool,
+    pub id_only: bool,
```

Comment:
since we now have an `id` (prev `urn`), we changed this to `--id-only`

Reply:
`--id` is still applicable, as it'll return the id and nothing else.

Reply:
running `rad inspect` with no flag would return id as well, see [this](https://github.com/radicle-dev/heartwood/pull/99#discussion_r1020086805).

Reply:
Yes but we reserve the right to change the default behavior, whereas `--id` will never change.